tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -31,16 +31,16 @@ def cattle_url():
 
 
 @pytest.fixture(autouse=True, scope='session')
-def cleanup(client):
+def cleanup():
     to_delete = []
-    for i in client.list_instance(state='running'):
+    for i in _admin_client().list_instance(state='running'):
         try:
```

Comment:
Yes. _admin_client() tries to authenticate with a credential of admin and adminpass where as client() just straight up tries to make a client and will fail if auth is on. _admin_client() won't fail if auth is on and there is an apiKey for admin / adminpass. This will all need to be redone in my follow up pr for auth after projects is merged. Since all of the client fixtures will be redone any way.